### ISSUE TITLE
cant run make pipenv-install

### ISSUE THREAD

**Issue by angelss197200** [Guilhem23/Jackett_Indexerr#6]
my process is cd to the extracted files folder and run "make pipenv-install"

i get this

> if [ -f Pipfile.lock ]; then \
        pipenv install --dev --ignore-pipfile; \
else \
        pipenv install --dev --skip-lock; \
fi;
Installing dependencies from Pipfile.lock (193d8a)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/0 — 00:00:00
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
pipenv run pip install -e .
Obtaining file:///home/angel/Documents/Jackett_Indexerr-master
    ERROR: Command errored out with exit status 1:
     command: /home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/home/angel/Documents/Jackett_Indexerr-master/setup.py'"'"'; __file__='"'"'/home/angel/Documents/Jackett_Indexerr-master/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-hkh3_val
         cwd: /home/angel/Documents/Jackett_Indexerr-master/
    Complete output (16 lines):
    ERROR:root:Error parsing
    Traceback (most recent call last):
      File "/home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/lib/python3.8/site-packages/pbr/core.py", line 96, in pbr
        attrs = util.cfg_to_args(path, dist.script_args)
      File "/home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/lib/python3.8/site-packages/pbr/util.py", line 271, in cfg_to_args
        pbr.hooks.setup_hook(config)
      File "/home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/lib/python3.8/site-packages/pbr/hooks/__init__.py", line 25, in setup_hook
        metadata_config.run()
      File "/home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/lib/python3.8/site-packages/pbr/hooks/base.py", line 27, in run
        self.hook()
      File "/home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/lib/python3.8/site-packages/pbr/hooks/metadata.py", line 25, in hook
        self.config['version'] = packaging.get_version(
      File "/home/angel/.local/share/virtualenvs/Jackett_Indexerr-master-ebT4FoWd/lib/python3.8/site-packages/pbr/packaging.py", line 870, in get_version
        raise Exception("Versioning for this project requires either an sdist"
    Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name Jackett_Indexerr was given, but was not able to be found.
    error in setup command: Error parsing /home/angel/Documents/Jackett_Indexerr-master/setup.cfg: Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. It's also possible that there is a mismatch between the package name in setup.cfg and the argument given to pbr.version.VersionInfo. Project name Jackett_Indexerr was given, but was not able to be found.
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
make: *** [Makefile:108: pipenv-install] Error 1

**Comment by RomanSoldeirs** [comment]
I banged my head against this exact problem for 3 hours today. Turned out to be the way i got the files. If you download the files as a zip then extract them you get the error above. The following worked for me.

_apt-get install git
git clone https://github.com/Guilhem23/Jackett_Indexerr.git
cd Jackett_Indexerr
make pipenv-install_
_nano config.ini_ (put jackett and sonarr setting in)
_make run_

Hope that helps you or anyone else that stumbles by this
